lib/Optimizer/GraphOptimizer/PassManager.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Optimizer/GraphOptimizer/PassManager.h"
+
+#include "glow/Optimizer/GraphOptimizer/FunctionPasses.h"
+
+#include <glog/logging.h>
+
+using namespace glow;
+
+bool FunctionPassManager::runPrePass(const FunctionPass &P) { return false; }
```

Comment:
`runPrePass` and `runPostPass` could be good places to insert some dumping/logging before/after each pass.